python/modules/IcePy/Operation.cpp

```diff
@@ -61,7 +72,6 @@ namespace IcePy
         ParamInfoList optionalOutParams;
         ParamInfoPtr returnType;
         ExceptionInfoList exceptions;
-        string dispatchName;
```

Comment:
We use to store the computed mapped name in this field.
Now that we directly have the mapped name, this field is useless.